test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -69,6 +69,24 @@ TEST_F(AwsLambdaFilterTest, DecodingHeaderStopIteration) {
   EXPECT_EQ(Http::FilterHeadersStatus::StopIteration, result);
 }
 
+TEST_F(AwsLambdaFilterTest, HostHeaderLambdaTrue) {
```

Comment:
Do you want to rename the test name to something like "HostHeaderSanitized", "HostHeaderNotSanitized"?